### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#634] [Q&A] [answered]
How to create if then else type more block
Like command block

**Comment by j-px** [comment, marked as answer]
Use Block Manager to create Blocks. There you can set the new Block's type to an if-Block (c).

**Reply by subhash16** [reply]
I know that but I want it in more block,is it possible

**Reply by j-px** [reply]
Oh, I overlooked that part, sorry.

More Blocks can only be regular Blocks, so no, sadly not. We can't change that just yet, we haven't been able to modify the Block logic yet.

**Comment by subhash16** [comment]
Ok thanks
